awx/ui/src/screens/Instances/InstanceDetail/InstanceDetail.js

```diff
@@ -288,8 +296,14 @@ function InstanceDetail({ setBreadcrumb }) {
           />
         )}
       </DetailList>
-      {!isHopNode && (
+      {!isHopNode && me.is_superuser && isK8s && (
```

Comment:
Adding this check here would make it so that non-k8s users can't run Health Checks on their Instances.